.eslintrc.yaml

```diff
@@ -98,11 +98,18 @@ rules:
   func-call-spacing: 2
   func-name-matching: 2
   func-style: [2, declaration, {allowArrowFunctions: true}]
-  indent: [2, 2, {ArrayExpression: first,
-                  CallExpression: {arguments: first},
-                  MemberExpression: 1,
-                  ObjectExpression: first,
-                  SwitchCase: 1}]
+  # indent: [2, 2, {ArrayExpression: first,
```

Comment:
Yes, that's right.

We will probably want to keep the legacy rules in the doc directory until the next ESLint release because the bug in https://github.com/eslint/eslint/commit/ab8b0167bdaf3b8851eab3fbc2769f2bdd71677b affects a lot of our example code.

But everything else can move over to the new rules sooner if we want.